integration-tests/package.json

```diff
@@ -15,12 +15,14 @@
     "@aws-sdk/client-s3": "3.693.0",
     "@esm2cjs/execa": "6.1.1-cjs.1",
     "@graphql-hive/apollo": "workspace:*",
+    "@graphql-hive/cli": "workspace:*",
```

Comment:
It would be great to get our coverage to _not_ be child-process based because each CLI invocation is taking 1-2 seconds which is terrible. Ideally we can reserve child process spawning tests for only a select few cases that really need that. I briefly played around with that while waiting for CI a few times. Currently this dep remains unused, we could remove.